Add guides to meet most of GitHub Community Standards (#6)
SRGSSR/guilde-plateformes-propres: docs/guides/CODE_OF_CONDUCT_GUIDE.md

Reviewer: Maybe we could also add a link to https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project?

# Code of Conduct Guide

Promote a welcoming and safe environment for discussions and contributions.

## Picking a Code of Conduct

> The [Contributor Covenant](https://www.contributor-covenant.org/) provides a great Code of Conduct. Except if you want to write your own you should simply adopt it.

Reviewer: ```suggestion
The [Contributor Covenant](https://www.contributor-covenant.org/) provides a great Code of Conduct. You can use it as-is, adapt it to your needs, or write your own document.
```

## Adding a Code of Conduct

> Add a file called `CODE_OF_CONDUCT.md` to your repository with the text of the Code of Conduct you chose. GitHub automatically adds a link to this file in the _About_ section of your repository homepage.

Reviewer: ```suggestion
Add a file called `CODE_OF_CONDUCT.md` to the root of your repository, the `docs` folder, or the `.github` folder, with the text of the Code of Conduct you chose. GitHub automatically adds a link to this file in the _About_ section of your repository homepage.
```
